### PR TITLE
Document updates for the term threshold changes

### PR DESCRIPTION
**Pull request by khansaad**
This PR adds a new MD file describing the Term threshold changes going as part of the PR #1081

### CODE REVIEW

**Review by msvinaykumar — APPROVED** [review]

File: design/TermThresholdDesign.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Term Threshold
+Presently, recommendations are categorized into Short, Medium, and Long terms based on the duration of historical data considered.
+
+Earlier we needed at least 24 hours data for Short term to be able to generate recommendation To ensure data adequacy for generating recommendations, we have introduced Minimum Data Thresholds:
```
Comment (chandrams) [inline design/TermThresholdDesign.md:4]: Minor typo, add a full stop after generate recommendation

after generate recommendation => generate recommendations.

File: design/TermThresholdDesign.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Term Threshold
+Presently, recommendations are categorized into Short, Medium, and Long terms based on the duration of historical data considered.
```
Comment (chandrams) [inline design/TermThresholdDesign.md:2]: You can add the issue due to this and mention about enforcing the term duration here

**Review by chandrams — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**